modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/api.js

```diff
@@ -41,4 +41,42 @@ const getHighPriorityContacts = async (filter) => {
 	return response.json();
 };
 
-export {getHighPriorityContacts, fetchHeadless};
+const getTicketAttachments = async (search) => {
+	// eslint-disable-next-line @liferay/portal/no-global-fetch
+	const response = await fetch(
+		`${HEADLESS_BASE_URL}${`c/ticketattachments/?search=${search}`}`,
+		{
+			headers: {
+				'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+				'Cache-Control': 'max-age=30, stale-while-revalidate=30',
+				'x-csrf-token': Liferay.authToken,
+			},
+		}
+	);
+
+	return response;
+};
+
+const deleteTicketAttachment = async (ticketAttachmentId) => {
+	// eslint-disable-next-line @liferay/portal/no-global-fetch
+	const response = await fetch(
+		`${HEADLESS_BASE_URL}${`c/ticketattachments/${ticketAttachmentId}`}`,
+		{
+			headers: {
+				'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+				'Cache-Control': 'max-age=30, stale-while-revalidate=30',
+				'x-csrf-token': Liferay.authToken,
+			},
+			method: 'DELETE',
+		}
+	);
+
+	return response;
+};
```

Comment:
Hi @liaoethan could you use `fetcher` instead of `fetch` ?

We have this function with all this configured